MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXPaymentCongratsViewModel.swift

```diff
@@ -239,6 +240,13 @@ extension PXPaymentCongratsViewModel: PXNewResultViewModelInterface {
     func getBottomCustomView() -> UIView? {
         return paymentCongrats.bottomView
     }
+    
+    func selectPaymentMethodConsumerCredits() -> Bool {
```

Comment:
Acho que fica mais enxuto assim, repare que a condição dentro do if já é a resposta que queremos ;)
```suggestion
    var isConsumerCredits: Bool {
     paymentResult.paymentMethodId == PXPaymentTypes.CONSUMER_CREDITS.rawValue
    }
```

Reply:
feito